src/main/scala/Base2Conversion.scala

```diff
@@ -0,0 +1,34 @@
+import scala.annotation.tailrec
+
+object Base2Conversion {
```

Comment:
needs rename, get rid of the 2, you can do this by right clicking here in IntelliJ and then Refactor > Rename (this will rename all calls to this as well, which makes it better than just changing everything manually)